world/office/scenes/fishing.py

```diff
@@ -89,7 +89,7 @@ def _print_records(self, fishing_state: FishingState):
             print(f"[info]Streak record: {fishing_state.record_streak} fish in a row")
 
     def _get_available_fish(self, fishing_state: FishingState) -> list[FishSpecies]:
-        available_fish = regular_fish
+        available_fish = [fish for fish in regular_fish]
```

Comment:
By inadvertently mutating the regular_fish array at line 94, Big Chungus could spawn in cases where the max_streak record exceeded the Big Chungus threshold, but the current_streak did not.